public/scripts/app.js

```diff
@@ -2,6 +2,144 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-// your code
+    //  COUNTER CLOCK - RESETS EVERYTIME PAGE IS REFRESHED
+    let startDateTime = new Date();
+    let startStamp = startDateTime.getTime();
 
+    function updateClock() {
+        let newDate = new Date();
+        let diff = Math.round(newDate-startStamp);
+        let days = Math.floor(diff/(1000*60*60*24));
+        let hours = Math.floor(diff/(1000*60*60));
+        let minutes = Math.floor(diff/(1000*60));
+        let seconds = Math.floor(diff/1000);
```

Comment:
`let seconds = Math.floor(diff/1000) % 60;` (and similar for the other time calculations)